src/converters/utils/preparePullRequestTimeline.ts

```
@@ -28,7 +28,10 @@ export const preparePullRequestTimeline = (
       review.user?.login !== pullRequestInfo?.user?.login &&
       checkUserInclusive(review.user?.login || invalidUserLogin)
   );
-  const approveTime = getApproveTime(pullRequestReviews);
+  const approveTime = getApproveTime(
+    pullRequestReviews,
+    parseInt(getValueAsIs("REQUIRED_APPROVALS"))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)